firecracker/microvm.py

```diff
@@ -278,6 +284,34 @@ def enable_rootfs(self, path_on_host: str) -> str:
         else:
             return path_on_host
 
+    def mount_rootfs(self, path_on_host: str) -> str:
```

Comment:
Using `pathlib.Path` for paths makes them more convenient to manipulate.

It also includes handy tools to concatenate paths such as `Path('/dir') / "filename")` and `Path('hello').symlink_to('world').